cypress/integration/internal/sroc-charge-version-flow.spec.js

```diff
@@ -41,89 +45,223 @@ describe('Create SRoC Charge version workflow journey', () => {
       cy.get('.govuk-button').contains('Set up a new charge').click();
     });
 
-    describe('user selects reason for new charge information', () => {
-      cy.get('.govuk-heading-l').contains('Select reason for new charge information');
-      cy.get('[type="radio"]#reason-12').click();
-      cy.get('button.govuk-button').click();
+    describe('Select reason for new charge information', () => {
```

Comment:
Describe relates to the page being loaded with a reference to the page title. 

Should allow easier reading, collapsing in a text editor to make the file smaller. 

can be one line:  
validateRadioOptions('Select reason for new charge information', 'reason-12', 'Select a reason for new charge information');

Reply:
https://design-system.service.gov.uk/patterns/question-pages/

All Page titles should be unique. 

Introducing them in this way allows us to establish un expected reuse of the title and to wrap each page to be tested in a describe block with the title as a unique key.